test/quicer_snb_SUITE.erl

```diff
@@ -2138,6 +2276,22 @@ wait_for_die([Pid | T]) ->
       wait_for_die(T)
   end.
 
+%% @doc find the starting point of the test run
+%%  some GC test may hit the Rid from previous run
+flush_previous_run([], _StartingPointFun) ->
+  %%% Oops, maybe wrong starting point
+  [];
+flush_previous_run([StartingPoint | T], StartingPoint) ->
+  T;
+flush_previous_run([Event | T], StartingPointFun) when is_function(StartingPointFun) ->
```

Comment:
Just a note: this kinda reminds me of `?split_trace_at` from snabbkaffe.  :thinking:

Reply:
thanks I will check that!